package.json

```diff
@@ -1,12 +1,36 @@
 {
-  "name": "leaderboard",
-  "version": "1.0.0",
+  "author": "",
+  "dependencies": {
+    "lodash": "^4.17.21",
+    "path": "^0.12.7"
+  },
   "description": "",
-  "main": "index.js",
-  "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+  "devDependencies": {
+    "babel-eslint": "^10.1.0",
+    "css-loader": "^6.6.0",
+    "eslint": "^7.32.0",
+    "eslint-config-airbnb-base": "^14.2.1",
+    "eslint-plugin-import": "^2.25.4",
+    "html-webpack-plugin": "^5.5.0",
+    "jest": "^27.5.1",
+    "style-loader": "^3.3.1",
+    "stylelint": "^14.4.0",
+    "stylelint-config-standard": "^24.0.0",
+    "stylelint-csstree-validator": "^2.0.0",
+    "stylelint-scss": "^4.1.0",
+    "webpack": "^5.68.0",
+    "webpack-cli": "^4.9.2",
+    "webpack-dev-server": "^4.7.4"
   },
```

Comment:
Well done! All your webpack related packages are listed as a devdependency 😎

Reply:
It wasn't easy, to figure out how do this property. Thanks.